src/main/java/com/contentful/java/cda/CDAClient.java

```diff
@@ -228,7 +266,13 @@ public CDASpace fetchSpace() {
     return Observable.just(contentType);
   }
 
-  /** Returns a {@link CDAClient} builder. */
+  String createUserAgent() {
+    return String.format("contentful.java/%s", Util.getProperty("version.name"));
```

Comment:
Sounds like a good idea. I'll take a look on how to determine the current ios version ... ;)